packages/sit-onyx/src/components/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -29,10 +28,10 @@ defineSlots<{
 
 const { t } = injectI18n();
 
-const shouldShowExternalIcon = computed(() => {
-  if (props.withExternalIcon !== "auto") return props.withExternalIcon;
-  return isExternalLink(props.href ?? "");
-});
+const shouldShowExternalIcon = (args: OnyxNavItemProps) => {
+  if (args.withExternalIcon !== "auto") return args.withExternalIcon ?? false;
+  return isExternalLink(args.href ?? "");
+};
```

Comment:
```suggestion
const shouldShowExternalIcon = (args: OnyxNavItemProps) => {
  const withExternalIcon = args.withExternalIcon ?? "auto";
  if (withExternalIcon !== "auto") return args.withExternalIcon;
  return isExternalLink(args.href ?? "");
};
```

The default value defined by `withDefaults(defineProps<>())` does not work now for the children so we need to explicitly set it here